src/WorkflowManager/WorkflowManager/Validators/WorkflowValidator.cs

```diff
@@ -148,13 +162,18 @@ private static void DetectUnreferencedTasks(TaskObject[] tasks, TaskObject first
             }
         }
 
-        private static void ValidateWorkflowSpec(Workflow workflow)
+        private async Task ValidateWorkflowSpec(Workflow workflow)
         {
-            if (string.IsNullOrWhiteSpace(workflow.Name))
+            if (string.IsNullOrWhiteSpace(workflow.Name) is true)
```

Comment:
Ny reason why we added `is true` here?

Reply:
if that's the case then ignore the other comments regarding this type of change.

Is this something we could enforce as a code style?

Reply:
As in have some form of rule that could pick up when `is true` isn't being used?